### ISSUE TITLE
ValueError: _table isn't defined - the Table Metaclass should set it.

### ISSUE THREAD

**Issue by AmazingAkai** [piccolo-orm/piccolo#936]
```py
class Poll(Table, tablename="polls"):
    id = Serial(primary_key=True)
    author_id = BigInt()
    channel_id = BigInt()
    message_id = BigInt(null=True)
    question = Varchar()
    options = Array(Varchar())
    votes = Array(Array(BigInt()))
```

Getting same error as #500 on running migrations, probably due to this:
```py
votes = Array(Array(BigInt()))
```

**Comment by dantownsend** [comment]
It looks like the problem is because it's an array of arrays.

`Array(Array(Integer()))` works OK, but the reason `BigInt` fails is because it needs to know whether it is being inserted into SQLite / Postgres / CockroachDB, to determine the SQL type to use.

We currently have this code, which sets up the array columns:

https://github.com/piccolo-orm/piccolo/blob/4f66bd89e6add8a2d78392b69a0d3e4cecd88222/piccolo/table.py#L305-L306

It needs to detect if it's an array of arrays. It should be a fairly simple fix.

However, I don't think arrays of arrays work too well in Piccolo Admin - so there may be some other fixes involved there.

**Comment by dantownsend** [comment]
@AmazingAkai This should be fixed in `piccolo==1.3.2`. I'll create a separate issue on the Piccolo Admin repo to investigate how well nested arrays work there.
